README.md

```diff
@@ -156,7 +306,13 @@ Copy this value and add it to the file `functions/secrets.js`.
 const STRIPE_ENDPOINT_KEY = `${YOUR_LOCAL_SIGNING_SECRET_KEY}`;
 ```
 
-## Our Git flow
+
+---
+<!-- section 4 -->
```

Comment:
Why adding these?

Reply:
They were already in there breaking up Part 1. I assumed for some kind of markdown section breaks or something but if they have no purpose we can just remove altogether

Reply:
The —- will be a horizontal rule. Don’t think the comments do anything but